core-services/sanitize-prow-jobs/_config.yaml

```diff
@@ -6068,9 +6068,6 @@ groups:
     - openshift-openshift-azure-infra-periodics.yaml
     - openshift-release-infra-periodics.yaml
     - infra-periodics-origin-release-images.yaml
-  build02:
```

Comment:
Nice. I want this too.
ci-tools build takes over 1h on build02.
This give it a chance to go on another cluster.

Reply:
> ci-tools build takes over 1h on build02.

My mistake.
https://prow.ci.openshift.org/job-history/gs/origin-ci-test/logs/branch-ci-openshift-ci-tools-master-images
ci-tools build has been on b01 all the time.

That is because

https://github.com/openshift/release/blob/5bd0b42cb9b984963c90ffae92334707524c237e/core-services/sanitize-prow-jobs/_config.yaml#L11279-L11291